### PR TITLE
Added extended PiHole statistics

### PR DESCRIPTION
**Pull request by bramceulemans**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

I found that just an "online" indicator was a bit empty. This might make the PiHole component a bit more insightful.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I've read & comply with the [contributing guidelines](https://github.com/bastienwirtz/homer/blob/main/CONTRIBUTING.md)
- [x] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers. 
- [x] I have made corresponding changes the documentation (README.md).
- [x] I've checked my modifications for any breaking changes, especially in the `config.yml` file

### CODE REVIEW

File: src/components/services/PiHole.vue [hunk 1]
```diff
@@ -42,6 +49,20 @@ export default {
...
     };
   },
+  computed: {
+    percentage: function () {
+      if (this.status) {
+        return this.status.ads_percentage_today.toFixed(1);
+      }
+      return "";
+    },
+    blocked: function () {
```
Comment (bastienwirtz) [inline src/components/services/PiHole.vue:59]: Unused ?

**Review by bastienwirtz — CHANGES_REQUESTED** [review]
Hi @bramceulemans!

Nice addition to the PiHole custom service, thanks for your contribution! Just a little thing but otherwise it look good :+1:

File: src/components/services/PiHole.vue (hunk 1 above)
Comment (bramceulemans) [reply src/components/services/PiHole.vue:59]: Oh yes, sorry! Forgot to remove that.

**Comment by bastienwirtz** [comment]
@bramceulemans I think you removed the wrong one!

**Comment by bramceulemans** [comment]
Think I'm too tired 😂

**Comment by bramceulemans** [comment]
Now for real, it's fixed!

**Review by bastienwirtz — APPROVED** [review]
Perfect :ok_hand: 
Thanks!

**Pull request merged**